### DISCUSSION TITLE
Debug branch?

### DISCUSSION THREAD

**Discussion by fkiller** [ScrapeGraphAI/Scrapegraph-ai#749] [Q&A]
Newbie here. I'm also a newbie of Python so I built a new branch to run directly with code without installing a pacakge so I can trace scrapegraphai code when I run examples or my own code. Is this right to do or do you have a better way? For me, requirements.txt lacks some packages to run and telemetry gave me an error because it tries to read the package's metadata, which doesn't exist while running without the pacakge. Let me know what you think.

My debug branch:
https://github.com/fkiller/Scrapegraph-ai/tree/debug

**Comment by VinciGit00** [comment]
Use the debugger instead
